rust/src/quic/parser.rs

```diff
@@ -286,7 +288,18 @@ impl QuicHeader {
                             return Err(nom7::Err::Error(QuicError::InvalidPacket));
                         }
                     }
+                } else if version == QuicVersion::V2 {
+                    match (first & 0x30) >> 4 {
+                        0x01 => QuicType::Initial,
+                        0x02 => QuicType::ZeroRTT,
+                        0x03 => QuicType::Handshake,
+                        0x00 => QuicType::Retry,
+                        _ => {
+                            return Err(nom7::Err::Error(QuicError::InvalidPacket));
+                        }
+                    }
                 } else {
+                    //if version == QuicVersion(1)
```

Comment:
remove

Reply:
why ?
Do you want me to phrase this comment in English rather than pseudo code ?

Reply:
prefer that ya, my eyes immediately flagged it as a debug artifact